docs/hosting-a-server-with-northstar/basic-listen-server.md

```diff
@@ -10,10 +13,12 @@ The ports you'll need to forward are `37015` UDP, and `8081` TCP by default, if
 
 Whether you're running a listen or dedicated server, you'll generally want to mess with the configuration at least a bit. While I do think the default configuration settings are pretty ok, being able to change your server's name or password, or increasing your server's tickrate is often something you'll want to do. Server configuration can be modified in the file `R2Northstar/mods/Northstar.CustomServers/mod/cfg/autoexec_ns_server.cfg`, which will be executed on server startup.\
 
-Additionally dedicated server settings and configs can be used on a listen server.\
-Dedicated server startup arguments can be placed into `Titanfall2/ns_startup_args.txt`\
-Dedicated convars can be placed into `R2Northstar/mods/Northstar.CustomServers/mod/cfg/autoexec_ns_server.cfg`
-The only difference between a listen and dedicated server is that a listen can only run while the host is in the match. 
+Additionally most dedicated server settings and configs can be used on a listen server.\
+Dedicated server startup arguments can't seem to be used, as the private match lobby overrides them. However most these settings can all be changed in the lobby prior to game start, rendering it a moot point. \
+The only arg that is not able to be replicated (I think) via the lobby screen is `max_players` You'd need a dedicated server if you want to change that.\
```

Comment:
Tbh, I haven't tested `max_players` on listen server either.

Reply:
Yeah, can't figure out a way to get that work without dedi args, which don't _seem_ to be working.